builder/builder-module.c

```diff
@@ -1298,13 +1318,22 @@ builder_module_build (BuilderModule  *self,
     autotools = TRUE;
   else if (!strcmp (self->buildsystem, "cmake-ninja"))
     cmake_ninja = TRUE;
+  else if (!strcmp (self->buildsystem, "simple"))
+    simple = TRUE;
   else
     {
       g_set_error (error, G_IO_ERROR, G_IO_ERROR_FAILED, "module %s: Invalid buildsystem: \"%s\"",
                    self->name, self->buildsystem);
       return FALSE;
     }
 
+  if (simple && !self->build_commands)
```

Comment:
> Should probably error if build-commands are set but build system isn't simple?

Or maybe just warn that they will be ignored?

> I also think build-system specific properties are less than ideal, but builder has already gone down that road.

Yeah, I was thinking maybe do something like:

```json
    "buildsystem": [
        "cmd1 arg arg",
        "cmd2 arg arg"
    ]
```

This would avoid the buildsystem-specific properties, but it would probably complicate the code quite a bit... Iin fact I'm not even sure that would be possible. ^_^